app/src/main/java/com/travelguide/fragments/SearchListFragment.java

```diff
@@ -0,0 +1,223 @@
+package com.travelguide.fragments;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.text.TextUtils;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.afollestad.materialdialogs.MaterialDialog;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseObject;
+import com.parse.ParseQuery;
+import com.travelguide.R;
+import com.travelguide.adapters.TripPlanAdapter;
+import com.travelguide.decorations.VerticalSpaceItemDecoration;
+import com.travelguide.helpers.ItemClickSupport;
+import com.travelguide.helpers.NetworkAvailabilityCheck;
+import com.travelguide.listener.OnTripPlanListener;
+import com.travelguide.models.TripPlan;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author kprav
+ *
+ * History
+ *   10/25/2015     kprav       Initial Version
+ */
+public class SearchListFragment extends TripBaseFragment {
+
+    private static final String TAG = TripPlanListFragment.class.getSimpleName();
```

Comment:
Change from `TripPlanListFragment.class.getSimpleName();` to `SearchListFragment.class.getSimpleName()`